### ISSUE TITLE
Cannot call use inside use any more

### ISSUE THREAD

**Issue by tpluscode** [chaijs/chai#1638]
I found that in v5 this is no longer possible

```js
/** type {ChaiPlugin} */
export default function(chai, utils) {
  chai.use(anotherPlugin)
}
```

`use` is no longer included in the argument. Can we add it to match what was possible with v4?

```diff
export function use(fn) {
  const exports = {
+   use,
    AssertionError,
    util,
    config,
    expect,
    assert,
    Assertion,
    ...should
  };

  if (!~used.indexOf(fn)) {
    fn(exports, util);
    used.push(fn);
  }

  return exports;
};
```

**Comment by 43081j** [comment]
Probably makes sense to avoid weird situations with multiple versions of chai (since if you didn't care about that, you would just import `use`?)

Let's see what @keithamus  or @koddsson thinks though

**Comment by tpluscode** [comment]
```js
const { use } = await import('chai')
use(jestSnapshotPlugin())
```

This appears to work but I'm not sure if making the plugin function async is exactly supported. And there may always be the risk of importing wrong chai in case of duplicate resolutions

**Comment by 43081j** [comment]
Yep, it would be a lot cleaner if you wrote your plugin as esm of course too 

I think you're right though and we should add this

**Comment by tpluscode** [comment]
The plugin is esm. But it includes another to use the extra assertions and I called `use` inside so that end user does not have to

**Comment by 43081j** [comment]
What I mean is if you did import `use`, you could just do it normally in a regular import statement

That would be pretty clean code but wouldn't work well if for some reason there's two chai running at once
